### ISSUE TITLE
About Us page links are not working

### ISSUE THREAD

**Issue by aefhm** [oasisprotocol/oasis-wallet-ext#46]
Low priority (non-blocking).

GitHub link and others are not working. Also, maybe putting the text on one line would be nicer.

<img width="368" alt="Screen Shot 2021-08-16 at 8 33 20 PM" src="https://user-images.githubusercontent.com/4441690/129649589-128f6ee9-7ec2-4d0e-8782-e90686b40d2c.png">

**Comment by lindalu1111** [comment]
@lvshaoping007 can you have "Privacy Policy" and "User Agreement" displayed in one line (vs. 2 currently)? Thanks!

**Comment by lvshaoping007** [comment]
what this mean ? merge on one line ? how about others ?

**Comment by aefhm** [comment]
I think they would all look nice on one line.

**Comment by pro-wh** [comment]
they mean how it currently shows up like this

```
Privacy
Policy

User
Agreement
```

and to change it to look like this

```
Privacy Policy

User Agreement
```

**Comment by peterjgilbert** [comment]
Also, "Github" should instead be "GitHub"

**Comment by lindalu1111** [comment]
Privacy Policy: https://oasisprotocol.org/privacy-policy
GitHub: https://github.com/oasisprotocol/oasis-wallet-ext/
Contact Us: info@oasisprotocol.org

**Comment by tjanez** [comment]
Also, we should use "Terms and Conditions" instead of "User Agreement".

**Comment by tjanez** [comment]
> Also, we should use "Terms and Conditions" instead of "User Agreement".

Fixed in #128.

**Comment by pro-wh** [comment]
we still need the terms and conditions link
